tracim/development.ini.base

```diff
@@ -228,6 +228,9 @@ email.reply.token = mysecuretoken
 email.reply.check.heartbeat = 60
 email.reply.use_html_parsing = true
 email.reply.use_txt_parsing = true
+# Lockfile path is required for email_reply feature,
+# it's just an empty file use to prevent concurrent access to imap unseen mail
+email.reply.filelock_path = %(here)s/email_fetcher.lock
```

Comment:
default path is probably not good. @buxx @inkhey look at other similar parameters (it's similar to cache stuff) and update the default location

Reply:
by the way, replace `email.reply.filelock_path` by `email.reply.fetcher_lockfile_path`.

Reply:
@lebouquetin You suggest value like ` %(here)s/data` or somethng like `/tmp/lock` ? When inkhey start to code the lock system, i suggest to him to use something like `%(here)s/email_fetcher.lock` insteand something like `/tmp/lock` to prevent conflict between multiple tracim instances on same host.

Reply:
`/tmp/lock` is probably conflict-ready ;) we have `%(here)/cache` which is for cache, we have `%(here)/data` which is for data, so probably something like `%(here)/tmp` for temporary content...